cudf/dummyarray.py

```diff
@@ -0,0 +1,65 @@
+"""Provides dummy array for strides calculations.
+"""
+# Author: Pearu Peterson
+# Created: Spetember 2018
+
+import sys
+import numpy as np
+
+if sys.platform.startswith('linux'):
```

Comment:
Why is this only possible on linux?

Reply:
Because, for instance, Windows does not have `/dev/zero` equivalent for using it in numpy.memmap. The dummy array is used for strides calculations and numpy.memmap('/dev/zero') provides an easy solution for that under Linux. numba has the required implementation, we can either copy or reuse it from numba to support other platforms.

Reply:
Ahh I see. Maybe you could put this in as a comment. Unless @kkraus14 or someone else asks for it, I don't think it is necessary to support other platforms in this PR.  This is big enough as it is :wink:

Reply:
We only support Linux in cuDF currently so there's no need to specify if it's Linux or not.